plugins/dashboard_plugin.py

```diff
@@ -167,35 +154,29 @@ def get_db_info(self):
 
         return None, None, None
 
-    def get_run_info(self, runs):
-        if len(runs) > 0:
-            run = runs[0]
-            run_date = run.execution_date
+    def _get_localized_time(self, date):
+        pst_time = date.astimezone(PACIFIC_TIMEZONE)
```

Comment:
Did you know that ordering date _strings_ is not reliable????? Leaving these as datetimes fixes the sort order problem.